devel/ci/integration/bodhi/fedmsg.py

```diff
@@ -0,0 +1,14 @@
+import socket
+config = {
+    'sign_messages': False,
+    'active': True,
+    # 'cert_prefix': 'bodhi',
+    # 'certnames': {'bodhi.{}'.format(socket.gethostname()): 'bodhi'},
+    # TODO: setup a container for the fedmsg relay
```

Comment:
There's an open pull request from @jeremycline that switches Bodhi to use fedora messaging - I wonder if we can coordinate the two pull requests? https://github.com/fedora-infra/bodhi/pull/2754